docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -58,6 +58,9 @@ testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
 > Inactive tabs and minimized browser windows switch to a lower resource consumption mode
 > where tests do not always execute correctly.
 
+If a browser stops responding while it executes tests, TestCafe restarts the browser and tries to continue tests in a new window.
+The test during which the browser stopped responding is reported as failed.
```

Comment:
The test will not be marked as failed in described case. TestCafe will try to restart it